sdk/cosmos/azure-cosmos-spark_3-1_2-12/src/test/scala/com/azure/cosmos/spark/SparkBridgeImplementationInternalSpec.scala

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.cosmos.spark
+
+import com.azure.cosmos.implementation.SparkBridgeImplementationInternal
+
+class SparkBridgeImplementationInternalSpec extends UnitSpec {
+  private[this] val rnd = scala.util.Random
+  //scalastyle:off multiple.string.literals
+
+  it should "parse multiple LSN formats" in {
+    val rawLsn = rnd.nextLong()
```

Comment:
NIT Ultimately this would find the regression - but I think it makes sense to use rnd.nextLong + smallest possible value (1) and maybe max value and then maybe x random values vs. just a single one to increase chances that regressions are found